src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -158,6 +150,15 @@ public Map<CreationEnum, List<? extends AzureAccount>> addAzureAccounts(
         return processedAccounts;
     }
 
+    private void checkAndAddToErrorAccount(Boolean checkCondition, AzureAccount azureAccount, String errorMessage,
+                                           List<ErroredAzureAccount> erroredAccounts) {
+        if (checkCondition.equals(Boolean.FALSE)) {
```

Comment:
There is no need to use a Boolean object here. The original boolean for emailSent is a primitive. You can just pass in the primitive lower case boolean. In that case, just need to use the condition condition for comparison "if (checkCondition == false)"